### ISSUE TITLE
Reasons to remove or keep clock as an event 

### ISSUE THREAD

**Issue by Samuel-Baird** [thewca/wca-regulations#848]
Here is a list of pros and cons for what happens if clock is removed as an event

Positive outcomes of removing Clock
* Clocks easily fall over or have their pin state messed up in transit so clocks being sent back to the scrambling table for extras is very common and takes up  competition time and results in delegates needing to approve a bunch of extras
* To fix the aforementioned issues usually needs special clock covers in order to avoid pressing in the pins. These cost organizers extra money and take up additional space in their storage. 
 * Clock can also be a pretty big hassle if no decent scramblers are available. This puzzle is very different from other WCA puzzles and as a result scrambling ability does not carry over nearly as well as other events Ex. someone who can scramble 3x3 can probably scramble NxN, feet, one-handed and 3bld pretty well. 
* Hardware has seen almost no improvement, the only clock currently being mass-produced was created in 2014. 
 * Hardware for clock can be extremely problematic from scramblers, out of the box lingaos (which are the only mass-produced clock) are extremely difficult to turn and alter the pin-state of. Once broken in more the pins can become the opposite, they fall down and have to be held up the entire time while scrambling (which is difficult). since the clock is a lot harder to set up than other puzzles and cubing retailers sell setup clocks far less often than other puzzles it results in a large number of clocks in many areas being un-setup. 
* Clock and feet are commonly requested for removal, removing it would satisfy the part of the community who have asked for it to be removed. 

Negative outcomes of removing Clock 

* Removing the event would allow no more official Clock solves, and all Clock Solves in future competitions would be unofficial and less likely to be held. This would disappoint those who enjoy participating in the event and those who enjoy watching it at competitions.

Neutral outcomes of removing Clock

* The popularity and appeal of Clock Solving are not the same in all countries. There are communities that love the event, and others that despise it. Its removal would satisfy some communities, but dissatisfy others.
* Clock adds very little to the WCA, the puzzle is very simple to solve and requires almost no knowledge to improve. Almost 100% of improvement is just practice in order to turn faster. There had been almost no development in methods since it was added to the WCA as an event. 
* Clock is not as enjoyable to watch as other events since it’s difficult for spectators to tell how much progress has been made and if the puzzle is solved or not. 
* Clock has one of the lowest competitor counts of speed solving events (FMC has more competitors too) Removing it wouldn’t affect too many competitors. 
* (Small positive but I’m going to put this here). Clock, unlike other puzzles, has no misalignment penalties and a less obvious solved state many judges (especially non-cuber volunteers, new competitors or inexperienced competitors) can get confused about clocks solved state or if it’s a +2 if it’s off by a single move, this results in the judge needing to get help from another judge or the delegate and slows down the competition. 
 * Clock is extremely different from other puzzles
* clock has a very high level of turning accuracy which means solvers often compete poorly or DNF

**Comment by TheoProferes** [comment]
While it is true that some communities find it hard to hold clock, this is mainly due to that communites inexperience with the event, and among other communities where it is more popular it is far easier to run, for example the UK, where we have it almost every competition and almost always without any issue. This same point can be applied to other events too, for example square-1. In communities just starting to host square-1 at competitions they will have issues with scrambling, but that doesn't mean it should be removed. 

While there clearly is a dislike of clock from the community as a whole, there are more haters of it then have ever competed in it, and from my experience at competitions most of the hate of it comes from younger children who don't even know how to solve a clock, but have been told to hate clock by a few online influencers, and have developed there opinion of it there.

I would say that clock is in fact one of the most unique events in the wca. For example, if I had 7x7 as a main event, and it is removed I can start maining 6x6 and still be at the top of the community for it, years of 7x7 practise would not go to waste. However, if I'm a clock main in contention for winning every competition, and it is removed, I have no carry over to other events.

While it is not too important to the discussion of removing clock, I have to disagree with your point about clock being a event where you only have to practise turning. Clock is one of the few events where we can see a near optimal solution, and being able to see that near optimal solution every solve and look ahead on it just as important as being able to turn fast. As well as this, the general opinion on it being that there is nothing to learn comes from the size of the community. Techniques like the secret method and building lucky cases are all hard to learn and execute, but hardly no one knows about them from the lack of attention that clock gets. 

In conclusion, while it is true that there is a lot of hate for clock, I believe it isn't real hate for the event, rather then just a idea of a few popular cubers which had spread. Organisers have no obligation to hold clock at competition, so keeping it would have no effect on the community, whereas removing it would be removing the favourite event of thousands of cubers, and potentially stopping them from ever really finding a event that they fit into again.

**Comment by SeanMoran27** [comment]
i definitely agree in the sense that there is a lot of ambiguity when it comes to judging, however I feel like most of the issues with 'it's difficult to hold' can be sorted by judges/runners actually paying attention while judging/running, and having good knowledge of the regs

this will probably sound weird but I see a lot of parallels between clock at the moment and Square-1 a few years ago.

Square-1 had shit hardware and it got to the point where you had to put paper into the slice to stop it turning during transit, but this became almost unnecessary come the release of Qiyi/Volt/Yuxin/all squans post 2015, so I think something similar could happen with clock hardware in the near future which would remove many issues with holding it, we just need to wait out for a company to produce one

**Comment by Burnsy101** [comment]
Sean I can see some of the hardware parallels between clock and Square one, but for me it almost feels like they end there. In 2015 pre-good squans, therewere still some developments being made, such as people learning more and more EP algs, as well as the concept CSP starting to come about. 

Its also important to remember that in 2015, hardware was worse for cubers overall and records were that much slower - only 13 people had a sub 3 7x7 mean compared to 200 today.

It is possible that there will be  a hardware jump in clock, and good clocks have been released, but im almost feeling pessimistic considering the release of a 'good clock' has been anticipated for years.

There arent as many possible method jumps in clock like there are in almost every other event -  the top Square-1 solves use CSP, PBL and some sort of OBL for a 3/4 look solve now - by comparison, all clock solves can be relatively easily done 100% efficiently and it doesn't seem like there can be jumps in methods.

Who knows, I could be wrong and in 2021 we have 100 sub 5 singles, one-looking with no flip and good clock, but im highly doubtful

**Comment by TheoProferes** [comment]
I disagree with your point about the method jumps , there are numerous improvements that you can make to your solves, 4 immediately come to mind, secret method, building lucky cases, over the shoulder and pseudo-lucky cases, all have which took ages to learn and implement and have made me way faster. The difference here is that two of those techniques I had to come up with myself, and one was shown to me by a friend. Unlike most events their is not a vast amount of information online on how to improve, but that makes improvement all the more rewarding because your inventing them yourself

**Comment by xsrvmy** [comment]
I think the two main reason competitors may not like clock are:
- DNFs happen too often
- clock is commutative so there is less of a challenge, and a fixed solution can be used

(FYI I do not think clock should be removed.)

**Comment by Samuel-Baird** [comment]
Here are a few polls from the private Facebook group cyoubx's friends on removing clock. 

Note: the voting results of the groups may not accurately reflect the overall opinions of all competitors. These polls do have some high turnouts though so that’s good :) 

August 2017
Remove: 250 (44.72%)
Keep: 171 (30.59%)
Don’t care: 138 (24.69%)
Total: 559

January 2018
Remove: 225 (44.82%)
Keep: 277 (55.18%)
Total: 502

December 2, 2018
Remove: 134 (41.36%)
Keep: 190 (58.64%)
Total: 324

December 3, 2018
Remove: 230 (53%)
Keep: 204 (47%)
Total: 434

December 9, 2018
Remove: 107 (37.54%)
Keep: 178 (62.46%)
Total: 285

August 2019
Keep: 220 (45.17%)
Remove: 200 (41.07%)
Undecided: 62 (12.73%)
Move to a lower tier: 5 (1%)
Total: 487

December 2019
Remove: 463 (62.9%)
Keep: 273 (37.1%)
Total: 736

Overall voting counts (this is counting votes from the some of the same people) 

1629 remove (48.96%)
1493 keep (44.88%) 
Don’t care/undecided/demote to lower tier 205 (6.16%) 

Total: 3327

![4A196C19-562F-4819-9E20-B658CE06C04D](https://user-images.githubusercontent.com/45566917/71454578-b07aed80-274e-11ea-9ec8-d05a43a93dfc.jpeg)
![075CA6D2-71DB-435C-910E-2E984BA7050C](https://user-images.githubusercontent.com/45566917/71454579-b07aed80-274e-11ea-87bb-68a7987e85f2.jpeg)
![F6ABB557-1E1B-4676-8435-E91558C492E8](https://user-images.githubusercontent.com/45566917/71454580-b1138400-274e-11ea-938f-fcf7faf129f7.jpeg)
![5DA0A2B5-8651-4C45-98B3-45FB38EAF7DE](https://user-images.githubusercontent.com/45566917/71454581-b1138400-274e-11ea-971e-a3442dc4b80e.jpeg)
![DD2D8FF1-A591-44B6-B029-EE6817C6D315](https://user-images.githubusercontent.com/45566917/71454583-b1138400-274e-11ea-8a10-16969bbcd031.jpeg)
![10D6A4BF-5E02-4D91-922A-E4C4B5569E9F](https://user-images.githubusercontent.com/45566917/71454584-b1138400-274e-11ea-892d-373e1d0e2453.jpeg)
![892627D2-D81A-4637-ACC5-5B493CA34C16](https://user-images.githubusercontent.com/45566917/71454585-b1138400-274e-11ea-9737-8a617c2c524e.jpeg)

**Comment by xsrvmy** [comment]
Honestly I don't see any significance in the jump this month, given that WCA just announced the removal of feet. It's basically just 50/50
